src/rdbms-connect/azext_rdbms_connect/_params.py

```diff
@@ -0,0 +1,32 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+
+from knack.arguments import CLIArgumentType
+from azure.cli.core.local_context import LocalContextAttribute, LocalContextAction
+
+
+def load_arguments(self, _):
+
+    for command_group in ('mysql', 'postgres'):
+        server_name_arg_type = CLIArgumentType(metavar='NAME',
+                                               help="Name of the server. The name can contain only lowercase letters, numbers, and the hyphen (-) character. Minimum 3 characters and maximum 63 characters.",
+                                               local_context_attribute=LocalContextAttribute(name='server_name', actions=[LocalContextAction.SET, LocalContextAction.GET], scopes=['{} flexible-server'.format(command_group)]))
+
+        administrator_login_arg_type = CLIArgumentType(metavar='NAME',
+                                                       local_context_attribute=LocalContextAttribute(name='administrator_login', actions=[LocalContextAction.GET, LocalContextAction.SET], scopes=['{} flexible-server'.format(command_group)]))
+
+        database_name_arg_type = CLIArgumentType(metavar='NAME',
+                                                 local_context_attribute=LocalContextAttribute(name='database_name', actions=[LocalContextAction.GET, LocalContextAction.SET], scopes=['{} flexible-server'.format(command_group)]))
+
+        with self.argument_context('{} flexible-server connect'.format(command_group)) as c:
+            c.argument('server_name', id_part=None, options_list=['--name', '-n'], arg_type=server_name_arg_type)
+            c.argument('administrator_login', arg_group='Authentication', arg_type=administrator_login_arg_type, options_list=['--admin-user', '-u'],
+                       help='The login username of the administrator.')
+            c.argument('administrator_login_password', arg_group='Authentication', options_list=['--admin-password', '-p'],
+                       help='The login password of the administrator.')
+            c.argument('database_name', arg_type=database_name_arg_type, options_list=['--database-name', '-d'], help='The name of a database.')
+            c.argument('interactive_mode', options_list=['--interactive'], action='store_true', help='Pass this parameter to connect to database in interactive mode.')
+            c.argument('querytext', options_list=['--querytext', '-c'], help='A query to run against the flexible server.')
```

Comment:
just curious, why the short for `--query_text` is `-c` instead of `-q`?

Reply:
Per PMs.  --query is a saved parameter in CLI, so just in case didn't want to do -q.  Can change to -q if that makes more sense?

Reply:
Sorry the --querytext is per PM guidance.  -c is the short command I chose based on --query being reserved.

Reply:
Sorry for all the comments.  Looks like -q is not reserved even though --query is? @evelyn-ys can you confirm? If so I'll go ahead an make the change from -c to -q.

Reply:
It is true that `--query` is reserved. But `-q` is available😉

Reply:
I have already approved. If you want to make the change, I can hold on the merge until you finish